cfgov/cfgov/settings/production.py

```diff
@@ -144,3 +144,12 @@
     LOGGING["loggers"]["mozilla_django_oidc"] = {
         "level": "INFO",
     }
+
+if DEPLOY_ENVIRONMENT == "container":
+    MIDDLEWARE += (
+        "whitenoise.middleware.WhiteNoiseMiddleware",
```

Comment:
Any reason why we wouldn't want to migrate to this in legacy as well? We could even add this to development, see https://whitenoise.readthedocs.io/en/latest/django.html#using-whitenoise-in-development.

Reply:
I don't know that there are any downsides to bringing it into legacy and dev, I was just trying to scope the change tightly to the containerized needs. 

I could certainly do this!

Reply:
Not to complicate, but, if that makes sense, would it be worth doing a small whitenoise PR to main to keep iterative changes small?

Reply:
Yup, I can do that.

Reply:
PR is #8678 

Will rebase and rework the commits on this branch once that is merged.